CONTRIBUTING.md

```diff
@@ -17,41 +16,62 @@ This guide will help you get started and outline some things you should know whe
   (install [directly](https://www.ruby-lang.org/en/documentation/installation),
   or via a ruby manager like [rvm](https://rvm.io/)
   or [rbenv](https://github.com/sstephenson/rbenv))
-  * Language the tests are written in
+  * language the tests are written in
 * [ShellCheck](https://github.com/koalaman/shellcheck)
-  * Used in the linting process to find common errors in the Bash code
+  * used in the linting process to find common errors in the Bash code
 
 
 ## Setup
 
+* install the [requirements](#requirements)
 * fork and clone the repository to your machine
-* `bundle` to install ruby gems
+* run `bundle` to install ruby gems
+* optionally run `rake` to make sure all tests pass on your machine
 
 
 ## Testing
 
-* Tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
-* All features need to have comprehensive test coverage
-* Source code and test files must pass the linters
+* tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
+* all features need to have comprehensive test coverage
+* source code and test files must pass the linters
 
 ```bash
-# rake tasks
-rake         # Run linters and feature tests
-rake format  # Run formatters (fixes some lint errors)
-rake lint    # Run linters
-rake test    # Run feature tests
+# running the different test types
+rake         # runs all tests
+rake lint    # runs the linters
+rake test    # runs the feature tests
+
+# running individual scenarios/features
+cucumber <filename>[:<lineno>]
+cucumber -n '<scenario/feature name>'
 
-# run single scenario/feature
-cucumber -n 'scenario/feature name'
-cucumber [filename][:lineno]
+# runnin   g individual scenarios/features while showing the application output
+DEBUG_COMMANDS=true cucumber <filename>[:<lineno>]
 
-# run single scenario/feature while showing the application output
-DEBUG_COMMANDS=true cucumber [filename][:lineno]
+# running several features in parallel
+bin/cuke <folder>
```

Comment:
folder is optional. And can technically just give a lot of files. bin/cuke is simply cucumber run in parallel